x/gov/types/keys.go

```diff
@@ -51,6 +51,9 @@ var (
 
 	// ParamsKey is the key to query all gov params
 	ParamsKey = []byte{0x30}
+
+	// KeyConstitution is the key string used to store the chain's constitution
+	KeyConstitution = []byte("constitution")
```

Comment:
Usually keys are numbers, but can't tell you exactlty what is the reason behind (key size?)

Reply:
Yes, key size